feature_layers/feature-collection-layer-from-portal/README.md

```diff
@@ -0,0 +1,34 @@
+# Feature Collection Layer From Portal
+
+This sample demonstrates opening a feature collection saved as a portal item
+
+![](FeatureCollectionLayerFromPortal.png)
+
+## Use case
+
+Feature collection layers are often used to hold features with mixed geometry or unstructured data. You can display feature collections stored in a Portal, which are often used for showing content from a CSV or map notes.
+
+## How to use the sample
+
+The feature collection is loaded from the Portal item when the sample starts. Enter feature collection saved as portal item in the text field
+and click on button to see result.
+
+## How it works
+
+To display a `FeatureCollection` as a `portal Item` on an `ArcGISMap` :
+
+1. Create a `Portal`.
+2. Create the `PortalItem`, referring to the portal and an item ID.
+3. Verify that the item represents a feature collection.
+4. Create a `FeatureCollection` from the item.
+5. Create a `FeatureCollectionLayer`, referring to the feature collection.
+6. Add the feature collection layer to the map's `OperationalLayers` collection.
+
+## Relevant API
+
+* FeatureCollection
+* FeatureCollectionLayer
+* Feature
+* Portal
+* PortalItem
+
```

Comment:
The readme is missing the 'about the data' section and the 'tags' from the common design:

```suggestion

## About the data

The sample uses a sample layer depicting [world populations](https://www.arcgis.com/home/item.html?id=32798dfad17942858d5eef82ee802f0b).

## Tags

collection, feature collection, feature collection layer, id, item, map notes, portal

```